cmd/locate.go

```diff
@@ -37,17 +36,42 @@ func init() {
 }
 
 func Locate(
-	buildlistFile string,
 	locatedFile string,
 ) error {
-	var buildlist []gotool.Module
-	err := ReadJSONFile(buildlistFile, &buildlist)
+	tidyCmd := exec.Command("go", "mod", "tidy")
+	err := tidyCmd.Run()
+	if err != nil {
+		return fmt.Errorf("running go mod tidy: %w", err)
+	}
+
+	vendorCmd := exec.Command("go", "mod", "vendor")
```

Comment:
Mod file gives us a list of our dependencies (e.g., github.com/blocky/prettyprinter) but not the URLs for fetching their source (e.g., https://github.com/blocky/prettyprinter.git). This is a problem because the third party license mining library tries to git fetch if the dependency is not located in the current working directory, but will fail since the dep names are not actual URLs.

I didn't know of an easy way for generating URLs from the mod name so decided to use `go mod vendor` for fetching the source code.